code/modules/mob/living/simple_animal/hostile/mushroom.dm

```diff
@@ -12,6 +12,7 @@
 	response_help  = "pets"
 	response_disarm = "gently pushes aside"
 	response_harm   = "whacks"
+	threat = 0.3
```

Comment:
All these critters do is attack each other in a highlander battle royale, and considering dozens of dozens of these shrooms can be spawned with hydroponics, this is a bit exploitable.